### DISCUSSION TITLE
Проблемы с MangaLib в 7 beta

### DISCUSSION THREAD

**Discussion by Meloni4** [lirix360/ReadmangaGrabber#42] [General]
В beta 7 скачивание с MangaLib не работает. При получении списка глав произошла ошибка. Подробности в лог-файле => [ERR]: (manga.go:32): Ошибка при получении списка глав: unexpected end of JSON input

**Comment by lirix360** [comment]
Поддержка MangaLib, как написано в релизе, эксперементальная, может сегодня работать, завтра нет, после завтра да.
